ClientSideKeyVaultKeyToCustomerProvidedKey/config.py

```diff
@@ -0,0 +1,31 @@
+from azure.storage.blob import CustomerProvidedEncryptionKey
+from azure.keyvault.keys.crypto import KeyWrapAlgorithm
+
+# the below values are fake and need to be replaced
+
+KEYVAULT_URL = ""
+CLIENT_ID = ""
+CLIENT_SECRET = ""
+TENANT_ID = ""
+connection_str = ""
+
+# items to make encryption scope
+SUB_ID = ""
+RESOURCE_GROUP = ""
+STORAGE_ACCOUNT = ""
+
+# if using customer provided key, set values here
+customer_key = CustomerProvidedEncryptionKey(key_value="",
+                                             key_hash="")
+
+# keyvault key used for encryption
+keyname = ""
+secret = ""
```

Comment:
>secret [](start = 0, length = 6)

what's the diff between this secret and CLIENT_SECRET?

Reply:
"secret" is used for the client side encryption using a key vault key. so if the key is using RSA it uses the "keyname" (changing this to CLIENTSIDE_KEYNAME), and if it is not using an RSA it uses "secret" (changing to KEYVAULT_SECRET). the CLIENT_SECRET is part of the service principal